modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -243,7 +243,7 @@ public List<ServiceRegistration<?>> deploy(
 					_resourcePermissionLocalService,
 					_userGroupRoleLocalService),
 				HashMapDictionaryBuilder.<String, Object>put(
-					"com.liferay.object", "true"
+					"com.liferay.object.definition", "true"
```

Comment:
Could you please check if we can use here the `ObjectConstants.RESOURCE_NAME_OBJECT_DEFINITION`?